node_registry.schema.json

```diff
@@ -157,12 +158,32 @@
           "type": "string",
           "minLength": 1
         },
-        "keywords": {
+        "types": {
           "type": "array",
           "minItems": 1,
           "items": {
             "type": "string",
-            "pattern": "^catalog|data|jupyterhub|other|service-(wps|wms|wfs|wcs|ogcapi_processes)$"
+            "enum": [
+              "auth",
+              "management",
+              "catalog",
+              "data",
+              "jupyterhub",
+              "other",
+              "wps",
+              "wms",
+              "wfs",
+              "wcs",
+              "ogcapi_processes",
+              "ogcapi_dggs"
```

Comment:
Should we be proactive and add the others as well?
(https://ogcapi.ogc.org/#standards)

I think `ogcapi_coverages` will be the next one on my roadmap. 

Wondering also if we should have `stac` explicitly or `catalog` is sufficient?
Technically, `stac` would be a `ogcapi_features` and `ogcapi_records` as well.